service languageServer{    
    string get_language(1:string text)
}    
